components/back-top/index.tsx

```diff
@@ -40,10 +40,6 @@ export interface BackTopProps {
 }
 
 export default class BackTop extends React.Component<BackTopProps, any> {
-  static defaultProps = {
-    visibilityHeight: 400,
-  };
```

Comment:
yes, I got it wrong...

It seems even if the props(typed in props interface: `a?: number`) has been defined in `defaultProps ` , for example:

```tsx
static defaultProps = {
	a: 1,
}
```  

I still need to specify the type when I use it?

```tsx
const { a } = this.props; 
const aa = a - 1;  // ERROR: prop 'a' possibly undefined
const add = (a as number) - 1;  //  ✔️
```